packages/splits-sdk/src/client/liquidSplit.ts

```diff
@@ -101,8 +100,10 @@ class LiquidSplitTransactions extends BaseTransactions {
     const nftAmounts = getNftCountsFromPercents(percentAllocations)
     const distributorFee = getBigIntFromPercent(distributorFeePercent)
 
+    const functionChainId = this._getFunctionChainId(chainId)
```

Comment:
in a scenario where a user passes in mainnet chain id wallet client but passes in base chain id in the createLiquidSplitConfig, would it silently create the split on mainnet?